apps/mediator/src/storage/StorageMessageQueue.ts

```diff
@@ -112,24 +113,50 @@ export class StorageServiceMessageQueue implements MessagePickupRepository {
   }
 
   private async sendNotification(agentContext: AgentContext, connectionId: string, messageType?: string) {
+    // Get the device token for the connection
+    const pushNotificationFcmRecord = await this.pushNotificationsFcmRepository.findSingleByQuery(agentContext, {
+      connectionId,
+    })
+
+    if (!pushNotificationFcmRecord?.deviceToken) {
+      this.agentContext.config.logger.info('No device token found for connectionId so skip sending notification')
+      return
+    }
+    // Check for firebase configuration
+    if (process.env.FIREBASE_PROJECT_ID) {
```

Comment:
I think we shouldn't mix the configuration and env variables. Can you add this to the configuration based on env values?